content/en/docs/setup/install/webhook/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Istio Webhook Management [Experimental]
+description: Shows how to manage webhooks in Istio through istioctl.
+weight: 10
+keywords: [security,webhook]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio has two webhooks (Galley and Sidecar Injector). In current Istio implementation,
+Galley and Sidecar Injector manage their own webhook configurations, which from
+security perspective is not recommended because a compromised webhook may conduct
+privilege escalation attacks.
+
+This task shows how to use `istioctl`, instead of Galley and Sidecar Injector, to
+manage the webhook configurations of Galley and Sidecar Injector.
+
+## Before you begin
+
+* Create a Kubernetes cluster with Istio installed. In the installation,
+[`global.operatorManageWebhooks`]({{< github_file >}}/install/kubernetes/helm/istio/values.yaml) should
+be set as true, and [DNS certificates should be configured](/docs/tasks/security/dns-cert).
+Istio installation guides can be found [here](/docs/setup/install).
+
+* Install [`jq`](https://stedolan.github.io/jq/) for json parsing.
+
+## Check webhook certificates
+
+The webhook certificates generated are stored in the secrets specified in the configuration.
+
+1.  Check that webhook certificates have been generated:
```

Comment:
No need to use a numbered list for only one item.
```suggestion
To display the DNS names in the webhook certificates of Galley and Sidecar Injector, run the following commands:
```

Reply:
Done.